modules/next/composer.json

```diff
@@ -17,6 +17,6 @@
     "drupal/subrequests": "^3.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^6.0 || ^7.0 || ^8.0 || ^9.0"
+    "phpunit/phpunit": "^8.0 || ^9.0 || ^10"
```

Comment:
@apathak18 I don't think we need phpunit 8, as I believe Drupal 10 works on PHPunit 9.5 and above.